core/operation/src/test/java/uk/gov/gchq/gaffer/operation/export/localfile/ExportToLocalFileTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.operation.export.localfile;
+
+import com.google.common.collect.Sets;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.Test;
+
+import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.jsonserialisation.JSONSerialiser;
+import uk.gov.gchq.gaffer.operation.OperationTest;
+import uk.gov.gchq.gaffer.operation.impl.export.localfile.ExportToLocalFile;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ExportToLocalFileTest extends OperationTest<ExportToLocalFile>  {
+
+    public static final String FILE_PATH = "path/to/file.csv";
+    public static final ArrayList<String> INPUT = Lists.newArrayList("header", "line1", "line2");
+
+    @Test
+    public void shouldJSONSerialiseAndDeserialise() throws SerialisationException {
+        // Given
+        ExportToLocalFile op = getPopulatedObject();
+
+        // When
+        final byte[] json = JSONSerialiser.serialise(op);
+        final ExportToLocalFile deserialisedOp = JSONSerialiser.deserialise(json, op.getClass());
+
+        // Then
+        assertThat(FILE_PATH).isEqualTo(deserialisedOp.getFilePath());
+        assertThat(INPUT).containsAll(deserialisedOp.getInput());
+    }
+
+    @Test
+    @Override
+    public void builderShouldCreatePopulatedOperation() {
+        // Given
+        ExportToLocalFile op = getPopulatedObject();
+
+        // Then
+        assertThat(FILE_PATH).isEqualTo(op.getFilePath());
+        assertThat(INPUT).containsAll(op.getInput());
+    }
+    @Test
+    @Override
+    public void shouldShallowCloneOperation() {
+        // Given
+        ExportToLocalFile op = getPopulatedObject();
+
+        // When
+        ExportToLocalFile clone = op.shallowClone();
+
+        // Then
+        assertThat(op).isNotEqualTo(clone);
+        assertThat(FILE_PATH).isEqualTo(clone.getFilePath());
+        assertThat(INPUT).isEqualTo(clone.getInput());
+    }
+
+    @Test
+    public void shouldGetOutputClass() {
+        // When
+        final Class<?> outputClass = getTestObject().getOutputClass();
+
+        // Then
+        assertThat(Object.class).isEqualTo(outputClass);
+    }
+
+    @Override
+    protected Set<String> getRequiredFields() {
+        return Sets.newHashSet("filePath");
```

Comment:
We should avoid using `Sets.newHastSet` as this is from Guava and [the docs for it mention](https://guava.dev/releases/22.0/api/docs/com/google/common/collect/Sets.html#newHashSet-E...-) the "method is not actually very useful and will likely be deprecated in the future".

It looks like `org.assertj.core.util.Sets` has an identical method. Although I think it would be better to use core Java.